docs/integrations/bullmq.md

```diff
@@ -12,24 +12,35 @@ BullMQ is a Node.js library that implements a fast and robust queue system built
 Since Dragonfly is highly compatible with Redis, BullMQ can be used with Dragonfly with zero or minimal code changes.
 By replacing Redis with Dragonfly, you can achieve superior performance and scalability for your BullMQ application.
 
+## TL;DR
+
+If you can use [hashtags](https://redis.io/docs/reference/cluster-spec/#hash-tags) in your queue names or prefixes (e.g., use `{queue1}` instead of `queue1`),
+execute the following command to run Dragonfly. This will enhance the performance of your BullMQ workloads.
```

Comment:
```suggestion
add the following flags. This will enhance the performance of your BullMQ workloads.
```
Does this clarify that other flags can be used as well?